__tests__/lib/context.test.js

```diff
@@ -0,0 +1,115 @@
+const { execSync } = require('child_process');
+
+const nock = require('nock');
+
+nock.disableNetConnect();
+
+const githubPayload = require('../__fixtures__/github-payload.json');
+
+describe('getContext()', () => {
+  // https://stackoverflow.com/a/48042799
```

Comment:
the solution below seems easier to maintain (and less code):

https://stackoverflow.com/a/58953365

in the interest of time - this should be fine

Reply:
I want to co-locate the process.env changes per test though, not globally across the whole suite. So I think what we have is best!